renku/core/commands/workflow.py

```diff
@@ -118,3 +124,110 @@ def _create_workflow(output_file, revision, paths, client: LocalClient):
 def create_workflow_command():
     """Command that create a workflow description for a file."""
     return Command().command(_create_workflow)
+
+
+@inject.autoparams()
+def _show_workflow(name_or_id: str, database: Database):
+    """Show the details of a workflow."""
+    workflow = database["plans"][name_or_id]
```

Comment:
```suggestion
    workflow = database["plans"].get(name_or_id)
```
because using the `operator[]` will actually raise a `KeyError` exception.
or of course surround it with try/catch